deploy/kubernetes/overlays/stable/kustomization.yaml

```diff
@@ -4,7 +4,7 @@ bases:
   - ../../base
 images:
   - name: gcr.io/k8s-staging-cloud-provider-ibm/ibm-powervs-block-csi-driver
-    newTag: main
+    newTag: v0.1.0-alpha.2
```

Comment:
I'm thinking of having 2 overlays like:
for staging `overlays/staging` for the prereleases
for production `overlays/stable` pointing to production images `gcr.io/cloud-provider-ibm/ibm-powervs-block-csi-driver`

Reply:
Concluded that 2 folders not needed.